src/Sulu/Bundle/PreviewBundle/Resources/js/views/Preview/stores/PreviewConfigStore.js

```diff
@@ -0,0 +1,46 @@
+// @flow
+import {action, computed, observable} from 'mobx';
+import queryString from 'query-string';
+
+export type PreviewRouteName = 'start' | 'render' | 'update' | 'update-context' | 'stop';
+export type PreviewMode = 'auto' | 'on_request' | 'off';
+
+type PreviewConfigStoreConfig = {
+    routes: { [PreviewRouteName]: string },
+    debounceDelay: number,
+    mode: PreviewMode,
+};
```

Comment:
We always put types into a separate `types.js` file, with the only exception of Props for react components.

Reply:
and add exact type `{| ... |}`

Reply:
this will be removed as soon as i refactor the store